include/Gaffer/ScriptNode.h

```diff
@@ -182,7 +182,7 @@ class ScriptNode : public Node
 		/// distinguishing between them.
 		bool isExecuting() const;
 		/// This signal is emitted following successful execution of a script.
-		typedef boost::signal<void ( ScriptNodePtr, const std::string )> ScriptExecutedSignal;
+		typedef boost::signal<void ( ScriptNode *, const std::string )> ScriptExecutedSignal;
```

Comment:
Unless we have a very good reason, we should omit this commit because it's an ABI change.

Reply:
Can you remind me what are the symptoms of the bug this was fixing? If needed I can make it a 0.36 release. Maybe it was a bug that wasn’t affecting IE but an external user?

Reply:
This commit was included in #2336, which fixed an expression copy/paste bug. I don't think the commit is part of the bug fix though, it's just something I fixed up along the way. From that PR :

>I've also included a fix for the ScriptNode::ScriptExecutedSignal signature because it tripped me up while writing this. This could be considered separately if you prefer.

When I said "tripped me up" I think I just meant "the signature was not what I expected it to be" rather than anything more important.

Reply:
I dropped the commit and rebased